<launch>

    <node name ="opt_laser_gps" pkg="laser_mapping" type="opt_laser_gps" output="screen">
    
        <param name="data_path" type="string" value="/media/dxq/D/data/bag/cherry/1205/15/laser_map"/>

    </node>

</launch>
